File: app/src/main/java/io/github/trumeen/bean/BmobCourseContentBean.java
package io.github.trumeen.bean;

import cn.bmob.v3.BmobObject;

public class BmobCourseContentBean extends BmobObject {

    public BmobCourseContentBean(){
        this.setTableName("course_content");
    }

    public String id;
    public String result;
}

File: app/src/main/java/io/github/trumeen/bean/BmobCourseContentBean.kt
package io.github.trumeen.bean

data class CourseContentBean(
    val results: List<BmobCourseContentResult>
)

data class BmobCourseContentResult(
    val createdAt: String,
    val id: String,
    val objectId: String,
    val result: String,
    val updatedAt: String
)